server/initializers/constants.ts

```diff
@@ -107,7 +107,7 @@ const JOB_CONCURRENCY: { [ id in JobType ]: number } = {
   'activitypub-http-fetcher': 1,
   'activitypub-follow': 3,
   'video-file-import': 1,
-  'video-file': 1,
+  'video-file': config.get<number>('transcoding.concurrency'),
```

Comment:
`CONFIG.TRANSCODING.CONCURRENCY`

Reply:
I'll have to copy the whole CONFIG block up higher, because my editor is saying that when I do this(`CONFIG.TRANSCODING.CONCURRENCY`), it's being used before declaration.

Reply:
I'lll move the `CONFIG` block to be right after the `LAST_MIGRATION_VERSION`, seeing as this makes more sense for if we change or introduce more variables later into the `CONFIG` and use the definitions in the constants